locale/basic_signs.de.tr

```diff
@@ -1,14 +1,14 @@
 # textdomain: basic_signs
 Locked Sign=Abgeschlossenes Schild
-Glass Sign=Glas Schild
-Obsidian Glass Sign=Obsidian Glas Schild
-Plastic Sign=Plastik Schild
+Glass Sign=Glasschild
+Obsidian Glass Sign=Obsidianglasschild
+Plastic Sign=Plastikschild
 green=grün
 yellow=gelb
 red=rot
-white_red=weißrot
-white_black=schwarzweiß
+white_red=weiß-rot
+white_black=weiß-schwarz
```

Comment:
IMO "schwarzweiss" is more common even though not a literal translation. Some languages prefer saying "black & white" others "white & black" it's not uncommon.

But it doesn't hurt me personally to see "weiss-schwarz"